src/SprykerMiddleware/Zed/Process/Communication/Plugin/Configuration/DefaultConfigurationProfilePlugin.php

```diff
@@ -17,6 +17,8 @@
 class DefaultConfigurationProfilePlugin extends AbstractPlugin implements ConfigurationProfilePluginInterface
 {
     /**
+     * @api
```

Comment:
Do we need inheritdoc for plugins same as for facades?

Reply:
We probably do at some point, yes.
Out of scope probably for the current sniffer release today.

Reply:
@dereuromark could you create an enhancement ticket for that?

Reply:
https://spryker.atlassian.net/browse/TE-111